src/components/RightSidebar/Participants/Participant.vue

```diff
@@ -1176,110 +1124,84 @@ export default {
 </script>
 
 <style lang="scss" scoped>
-.selected {
-	background-color: var(--color-primary-element-light);
-	border-radius: 5px;
-}
+.participant {
+	// Overwrite NcListItem styles
+	:deep(.list-item) {
+		overflow: hidden;
+		outline-offset: -2px;
+		cursor: default;
 
-.participant-row {
-	display: flex;
-	align-items: center;
-	cursor: default;
-	margin: 4px 0;
-	border-radius: var(--border-radius-pill);
-	height: 56px;
-	padding: 0 4px;
+		a,
+		a * {
+			cursor: default;
+		}
+	}
+
+	&--offline &__user-name {
+		color: var(--color-text-maxcontrast);
+	}
 
-	&.isSearched {
+	&--searched :deep(.list-item) {
 		cursor: pointer;
 	}
 
-	&__user-wrapper {
-		margin-top: -4px;
-		margin-left: 12px;
-		width: calc(100% - 100px);
-		display: flex;
-		flex-direction: column;
+	&.list-item__wrapper--active,
+	&.list-item__wrapper.active {
+		:deep(.list-item) {
+			background-color: var(--color-primary-element-light);
 
-		&.has-call-icon {
-			/** reduce text width to have some distance from call icon */
-			padding-right: 5px;
-			/** call icon on the right most column */
-			width: calc(100% - 90px);
-		}
+			&:hover,
+			&:focus-within,
+			&:has(:focus-visible),
+			&:has(:active) {
+				background-color: var(--color-primary-element-light-hover);
+			}
```

Comment:
As shown above, I am not sure if it is expected to have background-color on click focus, it seems like you can do some action on it from a header menu

Reply:
Better to be fixed in upstream. CUrrently `NcListItem` always supposes item to be a link.